bin/hoc-2015-survey/generate-organizer-list

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+require 'set'
+
+query = <<EOS
+SELECT email, name from forms WHERE kind='HocSignup2015' AND created_at > '2015-11-10 23:59:59' AND data LIKE '%"send_posters_flag_b":"1"%' AND processed_data LIKE '%"location_country_code_s":"US"%'
+EOS
```

Comment:
This is all in Pegasus, which uses Sequel, not ActiveRecord, and unfortunately no models. Improving this system so we have a standardized way to query and materialized views of contacts & lists is my top priority in the next sprint :)